### DISCUSSION TITLE
Yesterday URL

### DISCUSSION THREAD

**Discussion by mchristofides** [plausible/analytics#779] [Anything Else]
Hi there, nice work on Plausible, but I'm a bit addicted to checking analytics 😅

I tried to create a daily task to only check them once per day, but wanted to check the view for yesterday rather than for today (yesterday doesn't change, and I'm not tempted to check it again so soon, in theory at least).

Is there a way for me to get to the stats for "yesterday" via a relative URL? 

If not, it would be amazing if you could add one! I think the first thing I tried was the following, in case helpful: https://plausible.io/plausible.io?period=day&date=yesterday

**Comment by Vigasaurus** [comment]
There isn't; but to do this really quickly you could load into your dashboard and press the keys `D` and `Left Arrow`, this will switch you to a **D**ay-based view, and then go **Left** one day. This will always be n-1 to today, as `D` sets it to Today, and one day left would be Yesterday, naturally.

**Reply by mchristofides** [reply]
Thanks, that's helpful, at least on desktop

**Reply by metmarkosaric** [reply]
we made some changes to the date picker to accommodate the GA import so you can now also get to `Yesterday` directly with a keyboard shortcut. just press `E`. see all the other shortcuts here https://plausible.io/docs/keyboard-shortcuts
